catavatar/catavatar.php

```diff
@@ -18,48 +18,58 @@
 /**
  * Installs the addon hook
  */
-function catavatar_install() {
+function catavatar_install()
+{
 	Addon::registerHook('avatar_lookup', 'addon/catavatar/catavatar.php', 'catavatar_lookup');
 	Addon::registerHook('addon_settings', 'addon/catavatar/catavatar.php', 'catavatar_addon_settings');
 	Addon::registerHook('addon_settings_post', 'addon/catavatar/catavatar.php', 'catavatar_addon_settings_post');
 
-	logger("registered catavatar");
+	logger('registered catavatar');
 }
 
 /**
  * Removes the addon hook
  */
-function catavatar_uninstall() {
+function catavatar_uninstall()
+{
 	Addon::unregisterHook('avatar_lookup', 'addon/catavatar/catavatar.php', 'catavatar_lookup');
 	Addon::unregisterHook('addon_settings', 'addon/catavatar/catavatar.php', 'catavatar_addon_settings');
 	Addon::unregisterHook('addon_settings_post', 'addon/catavatar/catavatar.php', 'catavatar_addon_settings_post');
-	
-	logger("unregistered catavatar");
-}
 
+	logger('unregistered catavatar');
+}
 
-function catavatar_addon_settings(&$a, &$s) {
-	if(! local_user())
+/**
+ * Cat avatar user settings page
+ */
+function catavatar_addon_settings(&$a, &$s)
+{
+	if (!local_user()) {
 		return;
+	}
 
-	$t = get_markup_template("settings.tpl", "addon/catavatar/" );
+	$t = get_markup_template('settings.tpl', 'addon/catavatar/');
 	$s = replace_macros ($t, [
-		'$postpost' => x($_POST,"catavatar-morecat") || x($_POST,"catavatar-emailcat"),
+		'$postpost' => !empty($_POST['catavatar-morecat']) || !empty($_POST['catavatar-emailcat']),
 		'$uncache' => time(),
 		'$uid' => local_user(),
 		'$usecat' => L10n::t('Use Cat as Avatar'),
 		'$morecat' => L10n::t('More Random Cat!'),
 		'$emailcat' => L10n::t('Reset to email Cat'),
-		'$seed' => PConfig::get(local_user(), "catavatar", "seed", false),
-		'$header' => L10n::t('Cat Avatar').' '.L10n::t('Settings'),
+		'$seed' => PConfig::get(local_user(), 'catavatar', 'seed', false),
+		'$header' => L10n::t('Cat Avatar Settings'),
 	]);
-	return;
 }
 
-function catavatar_addon_settings_post(&$a, &$s) {
-	if(! local_user())
+/**
+ * Cat avatar user settings POST handle
+ */
+function catavatar_addon_settings_post(&$a, &$s)
```

Comment:
Obsolete: Please use the following argument list instead: `(\Friendica\App $a, &$s)`